<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="178R3F3" name="com.StudentPortal.StudentTests.StudentsDCRCAPITests" tests="141" failures="3" timestamp="2022-08-29T16:49:08 EDT" time="196.080" errors="24">
  <testcase name="LedgerCardPostaCharge" time="0.972" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="AddDocToStudent" time="0.923" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests">
    <failure type="java.lang.AssertionError" message="expected [200] but found [400]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [400]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:839)
at org.testng.Assert.assertEquals(Assert.java:849)
at com.StudentPortal.Pages.StudentRestApiPage.DocumentCreation(StudentRestApiPage.java:652)
at com.StudentPortal.Pages.StudentRestApiPage.AddDocumentToStudent(StudentRestApiPage.java:3144)
at com.StudentPortal.StudentTests.StudentsDCRCAPITests.AddDocToStudent(StudentsDCRCAPITests.java:627)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- AddDocToStudent -->
  <testcase name="TC163_PlaceStudentsIntoJob" time="0.619" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="DocumentScheduleCreation" time="0.852" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="AddDocListToStudent" time="0.720" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="CreateSubsidiary" time="0.654" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at io.restassured.path.json.JsonPath.getInt(JsonPath.java:244)
at com.StudentPortal.Pages.StudentRestApiPage.CreateSubsidiary(StudentRestApiPage.java:372)
at com.StudentPortal.StudentTests.StudentsDCRCAPITests.CreateSubsidiary(StudentsDCRCAPITests.java:578)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- CreateSubsidiary -->
  <testcase name="LedgerCardPostaCharge" time="0.831" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC163_PlaceStudentsIntoJob" time="0.753" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="AddDocListToStudent" time="0.794" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="AddSubsidiary" time="0.950" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="DocumentCreation" time="0.697" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests">
    <failure type="java.lang.AssertionError" message="expected [200] but found [400]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [400]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:839)
at org.testng.Assert.assertEquals(Assert.java:849)
at com.StudentPortal.Pages.StudentRestApiPage.DocumentCreation(StudentRestApiPage.java:652)
at com.StudentPortal.StudentTests.StudentsDCRCAPITests.DocumentCreation(StudentsDCRCAPITests.java:606)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- DocumentCreation -->
  <testcase name="AddLettertoEmp" time="0.740" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="DocumentPolicyCreation" time="1.751" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests">
    <failure type="java.lang.AssertionError" message="expected [200] but found [400]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [400]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:839)
at org.testng.Assert.assertEquals(Assert.java:849)
at com.StudentPortal.Pages.StudentRestApiPage.DocumentPolicy(StudentRestApiPage.java:3137)
at com.StudentPortal.StudentTests.StudentsDCRCAPITests.DocumentPolicyCreation(StudentsDCRCAPITests.java:620)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- DocumentPolicyCreation -->
  <testcase name="AddSubsidiary" time="0.666" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="AddSubsidiary" time="0.784" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC163_PlaceStudentsIntoJob" time="0.624" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="AddSubsidiary" time="0.556" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="LedgerCardPostaCharge" time="0.829" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="AddSubsidiary" time="0.532" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="AddDocListToStudent" time="1.954" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC248_1_PlacementStudentsIntoInternship" time="0.669" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC248_1_PlacementStudentsIntoInternship" time="0.589" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC248_1_PlacementStudentsIntoInternship" time="0.552" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="LedgerCardAdjustTransaction" time="1.723" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="LedgerCardAdjustTransaction" time="1.749" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="LedgerCardAdjustTransaction" time="1.875" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="LedgerChargeDelete" time="1.747" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC248_2_PlacementStudentsIntoExternship" time="0.609" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC248_2_PlacementStudentsIntoExternship" time="0.696" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="LedgerChargeDelete" time="1.476" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC248_2_PlacementStudentsIntoExternship" time="0.582" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="LedgerChargeDelete" time="1.858" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC216_1_AddFADocumentToIndividualStudents" time="0.880" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC216_1_AddFADocumentToIndividualStudents" time="1.015" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC216_1_AddFADocumentToIndividualStudents" time="0.773" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC216_2_AddFADocumentListToStudents" time="0.690" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC216_2_AddFADocumentListToStudents" time="0.871" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC216_2_AddFADocumentListToStudents" time="0.900" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC831_StudentApplicationCreation" time="0.999" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC835_TransferCredits" time="0.838" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="SubPostACharge" time="0.741" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC835_TransferCredits" time="0.871" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC835_TransferCredits" time="0.805" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC845_PostChargeToStudentLedgerCard" time="0.822" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="SubAdjustTransaction" time="1.563" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC847_PostPaymentForStudent" time="0.650" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.framework.util.TestReportsLog.takeScreenShot(TestReportsLog.java:116)
at com.framework.util.TestReportsLog.log(TestReportsLog.java:41)
at com.StudentPortal.Pages.APIAutomationPage.PostPayment(APIAutomationPage.java:591)
at com.StudentPortal.StudentTests.StudentsDCRCAPITests.TC847_PostPaymentForStudent(StudentsDCRCAPITests.java:96)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- TC847_PostPaymentForStudent -->
  <testcase name="TC840_PerformStatusChanges" time="0.625" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.framework.util.TestReportsLog.takeScreenShot(TestReportsLog.java:116)
at com.framework.util.TestReportsLog.log(TestReportsLog.java:41)
at com.StudentPortal.Pages.APIAutomationPage.StatusChange(APIAutomationPage.java:639)
at com.StudentPortal.StudentTests.StudentsDCRCAPITests.TC840_PerformStatusChanges(StudentsDCRCAPITests.java:104)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- TC840_PerformStatusChanges -->
  <testcase name="TC234_1_CreateStudentGroup" time="0.657" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC234_2_AddStudentToGroup" time="1.656" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.framework.util.TestReportsLog.takeScreenShot(TestReportsLog.java:116)
at com.framework.util.TestReportsLog.log(TestReportsLog.java:41)
at com.StudentPortal.Pages.APIAutomationPage.AddStudentToGroup(APIAutomationPage.java:745)
at com.StudentPortal.StudentTests.StudentsDCRCAPITests.TC234_2_AddStudentToGroup(StudentsDCRCAPITests.java:120)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- TC234_2_AddStudentToGroup -->
  <testcase name="TC833_1_AddCourseToStudentCourses" time="0.849" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC833_2_RegisterstudentforACourse" time="0.593" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.framework.util.TestReportsLog.takeScreenShot(TestReportsLog.java:116)
at com.framework.util.TestReportsLog.log(TestReportsLog.java:41)
at com.StudentPortal.Pages.APIAutomationPage.RegisterStudent(APIAutomationPage.java:931)
at com.StudentPortal.StudentTests.StudentsDCRCAPITests.TC833_2_RegisterstudentforACourse(StudentsDCRCAPITests.java:136)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- TC833_2_RegisterstudentforACourse -->
  <testcase name="TC834_DropCourseFromStudentSchedule" time="0.680" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.framework.util.TestReportsLog.takeScreenShot(TestReportsLog.java:116)
at com.framework.util.TestReportsLog.log(TestReportsLog.java:41)
at com.StudentPortal.Pages.APIAutomationPage.DropCourseFromStudentSchedule(APIAutomationPage.java:983)
at com.StudentPortal.StudentTests.StudentsDCRCAPITests.TC834_DropCourseFromStudentSchedule(StudentsDCRCAPITests.java:144)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- TC834_DropCourseFromStudentSchedule -->
  <testcase name="TC234_3_CreateRegistrationlocks" time="1.159" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC194_FAReassignmentToAY" time="0.737" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC836_RecordIndividualFinalGrades" time="0.628" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.framework.util.TestReportsLog.takeScreenShot(TestReportsLog.java:116)
at com.framework.util.TestReportsLog.log(TestReportsLog.java:41)
at com.StudentPortal.Pages.APIAutomationPage.RecordIndividualFinalGrades(APIAutomationPage.java:1415)
at com.StudentPortal.StudentTests.StudentsDCRCAPITests.TC836_RecordIndividualFinalGrades(StudentsDCRCAPITests.java:174)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- TC836_RecordIndividualFinalGrades -->
  <testcase name="TC848_PostFADisbursement" time="2.549" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC844_PostRefund" time="1.368" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC879_Addtaskstoastudent" time="1.093" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC879_Addtaskstoastudent" time="0.768" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC879_Addtaskstoastudent" time="0.913" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC882_ConfirmpendingTasksexistinContactManagerforstudent" time="1.073" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC882_ConfirmpendingTasksexistinContactManagerforstudent" time="0.941" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC882_ConfirmpendingTasksexistinContactManagerforstudent" time="1.380" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC882_ConfirmpendingTasksexistforStaff" time="0.989" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC882_ConfirmpendingTasksexistforStaff" time="0.973" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC882_ConfirmpendingTasksexistforStaff" time="1.121" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC883_ConfirmthatpendingTasksinContactManagercanbeeditedbyStudent" time="0.892" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC883_ConfirmthatpendingTasksinContactManagercanbeeditedbyStudent" time="0.884" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC883_ConfirmthatpendingTasksinContactManagercanbeeditedbyStudent" time="0.843" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC883_ConfirmthatpendingTaskscanbeeditedbyStaff" time="0.880" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC883_ConfirmthatpendingTaskscanbeeditedbyStaff" time="0.983" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC883_ConfirmthatpendingTaskscanbeeditedbyStaff" time="0.945" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC855_ReleaseToGeneralLedger" time="17.998" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC854_PerformRevenueRecognition" time="12.875" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC894_Validatetheabilitytoaddnewemployers" time="0.653" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC893_PlacestudentsintoanexternshipPosition" time="1.744" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC849_Scheduleastudentpaymentplan" time="2.509" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC849_Scheduleastudentpaymentplan" time="4.081" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests">
    <error type="java.net.ConnectException" message="Connection refused: connect">
      <![CDATA[java.net.ConnectException: Connection refused: connect
at java.net.DualStackPlainSocketImpl.connect0(Native Method)
at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79)
at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
at java.net.Socket.connect(Socket.java:606)
at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:121)
at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:180)
at org.apache.http.impl.conn.ManagedClientConnectionImpl.open(ManagedClientConnectionImpl.java:326)
at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:610)
at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:445)
at org.apache.http.impl.client.AbstractHttpClient.doExecute(AbstractHttpClient.java:835)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:56)
at org.apache.http.client.HttpClient$execute$0.call(Unknown Source)
at io.restassured.internal.RequestSpecificationImpl$RestAssuredHttpBuilder.doRequest(RequestSpecificationImpl.groovy:2129)
at io.restassured.internal.http.HTTPBuilder.post(HTTPBuilder.java:349)
at io.restassured.internal.http.HTTPBuilder$post$2.call(Unknown Source)
at io.restassured.internal.RequestSpecificationImpl.sendRequest(RequestSpecificationImpl.groovy:1271)
at sun.reflect.GeneratedMethodAccessor103.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:93)
at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:325)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1213)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1022)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:810)
at io.restassured.internal.RequestSpecificationImpl.invokeMethod(RequestSpecificationImpl.groovy)
at org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.call(PogoInterceptableSite.java:48)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:149)
at io.restassured.internal.filter.SendRequestFilter.filter(SendRequestFilter.groovy:30)
at io.restassured.filter.Filter$filter$0.call(Unknown Source)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:48)
at io.restassured.filter.Filter$filter.call(Unknown Source)
at io.restassured.internal.filter.FilterContextImpl.next(FilterContextImpl.groovy:72)
at io.restassured.filter.time.TimingFilter.filter(TimingFilter.java:56)
at io.restassured.filter.Filter$filter.call(Unknown Source)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:48)
at io.restassured.filter.Filter$filter$0.call(Unknown Source)
at io.restassured.internal.filter.FilterContextImpl.next(FilterContextImpl.groovy:72)
at io.restassured.filter.FilterContext$next.call(Unknown Source)
at io.restassured.internal.RequestSpecificationImpl.applyPathParamsAndSendRequest(RequestSpecificationImpl.groovy:1732)
at sun.reflect.GeneratedMethodAccessor96.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:93)
at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:325)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1213)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1022)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:810)
at io.restassured.internal.RequestSpecificationImpl.invokeMethod(RequestSpecificationImpl.groovy)
at org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.call(PogoInterceptableSite.java:48)
at org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.callCurrent(PogoInterceptableSite.java:58)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:182)
at io.restassured.internal.RequestSpecificationImpl.applyPathParamsAndSendRequest(RequestSpecificationImpl.groovy:1738)
at sun.reflect.GeneratedMethodAccessor95.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:93)
at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:325)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1213)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1022)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:810)
at io.restassured.internal.RequestSpecificationImpl.invokeMethod(RequestSpecificationImpl.groovy)
at org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.call(PogoInterceptableSite.java:48)
at org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.callCurrent(PogoInterceptableSite.java:58)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:182)
at io.restassured.internal.RequestSpecificationImpl.post(RequestSpecificationImpl.groovy:174)
at io.restassured.internal.RequestSpecificationImpl.post(RequestSpecificationImpl.groovy)
at com.StudentPortal.Pages.StudentTestsAPI.ScheduleStudentPaymentPlan_API(StudentTestsAPI.java:1548)
at com.StudentPortal.StudentTests.StudentsDCRCAPITests.TC849_Scheduleastudentpaymentplan(StudentsDCRCAPITests.java:282)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- TC849_Scheduleastudentpaymentplan -->
  <testcase name="TC849_Scheduleastudentpaymentplan" time="4.106" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests">
    <error type="java.net.ConnectException" message="Connection refused: connect">
      <![CDATA[java.net.ConnectException: Connection refused: connect
at java.net.DualStackPlainSocketImpl.connect0(Native Method)
at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79)
at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
at java.net.Socket.connect(Socket.java:606)
at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:121)
at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:180)
at org.apache.http.impl.conn.ManagedClientConnectionImpl.open(ManagedClientConnectionImpl.java:326)
at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:610)
at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:445)
at org.apache.http.impl.client.AbstractHttpClient.doExecute(AbstractHttpClient.java:835)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:56)
at org.apache.http.client.HttpClient$execute$0.call(Unknown Source)
at io.restassured.internal.RequestSpecificationImpl$RestAssuredHttpBuilder.doRequest(RequestSpecificationImpl.groovy:2129)
at io.restassured.internal.http.HTTPBuilder.post(HTTPBuilder.java:349)
at io.restassured.internal.http.HTTPBuilder$post$2.call(Unknown Source)
at io.restassured.internal.RequestSpecificationImpl.sendRequest(RequestSpecificationImpl.groovy:1271)
at sun.reflect.GeneratedMethodAccessor103.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:93)
at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:325)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1213)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1022)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:810)
at io.restassured.internal.RequestSpecificationImpl.invokeMethod(RequestSpecificationImpl.groovy)
at org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.call(PogoInterceptableSite.java:48)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:149)
at io.restassured.internal.filter.SendRequestFilter.filter(SendRequestFilter.groovy:30)
at io.restassured.filter.Filter$filter$0.call(Unknown Source)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:48)
at io.restassured.filter.Filter$filter.call(Unknown Source)
at io.restassured.internal.filter.FilterContextImpl.next(FilterContextImpl.groovy:72)
at io.restassured.filter.time.TimingFilter.filter(TimingFilter.java:56)
at io.restassured.filter.Filter$filter.call(Unknown Source)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:48)
at io.restassured.filter.Filter$filter$0.call(Unknown Source)
at io.restassured.internal.filter.FilterContextImpl.next(FilterContextImpl.groovy:72)
at io.restassured.filter.FilterContext$next.call(Unknown Source)
at io.restassured.internal.RequestSpecificationImpl.applyPathParamsAndSendRequest(RequestSpecificationImpl.groovy:1732)
at sun.reflect.GeneratedMethodAccessor96.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:93)
at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:325)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1213)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1022)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:810)
at io.restassured.internal.RequestSpecificationImpl.invokeMethod(RequestSpecificationImpl.groovy)
at org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.call(PogoInterceptableSite.java:48)
at org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.callCurrent(PogoInterceptableSite.java:58)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:182)
at io.restassured.internal.RequestSpecificationImpl.applyPathParamsAndSendRequest(RequestSpecificationImpl.groovy:1738)
at sun.reflect.GeneratedMethodAccessor95.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:93)
at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:325)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1213)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1022)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:810)
at io.restassured.internal.RequestSpecificationImpl.invokeMethod(RequestSpecificationImpl.groovy)
at org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.call(PogoInterceptableSite.java:48)
at org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.callCurrent(PogoInterceptableSite.java:58)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:182)
at io.restassured.internal.RequestSpecificationImpl.post(RequestSpecificationImpl.groovy:174)
at io.restassured.internal.RequestSpecificationImpl.post(RequestSpecificationImpl.groovy)
at com.StudentPortal.Pages.StudentTestsAPI.ScheduleStudentPaymentPlan_API(StudentTestsAPI.java:1548)
at com.StudentPortal.StudentTests.StudentsDCRCAPITests.TC849_Scheduleastudentpaymentplan(StudentsDCRCAPITests.java:282)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- TC849_Scheduleastudentpaymentplan -->
  <testcase name="TC888_ValidatetheabilitytopostFederalWorkStudydisbursements" time="0.835" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC887_ApprovingFADisbursementstoPayinBatchform" time="14.449" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.framework.util.TestReportsLog.takeScreenShot(TestReportsLog.java:116)
at com.framework.util.TestReportsLog.log(TestReportsLog.java:41)
at com.StudentPortal.Pages.StudentTestsAPI.ApprovingFADisbursements_API(StudentTestsAPI.java:1716)
at com.StudentPortal.StudentTests.StudentsDCRCAPITests.TC887_ApprovingFADisbursementstoPayinBatchform(StudentsDCRCAPITests.java:302)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- TC887_ApprovingFADisbursementstoPayinBatchform -->
  <testcase name="TC884_AbilitytoaddEstimatedFAPackagingpriortoEnrollment" time="0.807" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC880_ClosingthetasksfromtheContactManagerbyStaff" time="1.081" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC880_ClosingthetasksfromtheContactManagerbyStaff" time="0.911" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC880_ClosingthetasksfromtheContactManagerbyStaff" time="1.265" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="PlacestudentsintointernshipPosition" time="0.798" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="PlacestudentsintointernshipPosition" time="0.756" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="PlacestudentsintointernshipPosition" time="0.850" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="Abilitytoaddacoursesection" time="0.784" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.framework.util.TestReportsLog.takeScreenShot(TestReportsLog.java:116)
at com.framework.util.TestReportsLog.log(TestReportsLog.java:41)
at com.StudentPortal.Pages.DcrcApiPage.Abilitytoaddacoursesection(DcrcApiPage.java:231)
at com.StudentPortal.StudentTests.StudentsDCRCAPITests.Abilitytoaddacoursesection(StudentsDCRCAPITests.java:343)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- Abilitytoaddacoursesection -->
  <testcase name="Abilitytoaddacoursesection" time="0.689" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.framework.util.TestReportsLog.takeScreenShot(TestReportsLog.java:116)
at com.framework.util.TestReportsLog.log(TestReportsLog.java:41)
at com.StudentPortal.Pages.DcrcApiPage.Abilitytoaddacoursesection(DcrcApiPage.java:231)
at com.StudentPortal.StudentTests.StudentsDCRCAPITests.Abilitytoaddacoursesection(StudentsDCRCAPITests.java:343)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- Abilitytoaddacoursesection -->
  <testcase name="Abilitytoaddacoursesection" time="0.664" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.framework.util.TestReportsLog.takeScreenShot(TestReportsLog.java:116)
at com.framework.util.TestReportsLog.log(TestReportsLog.java:41)
at com.StudentPortal.Pages.DcrcApiPage.Abilitytoaddacoursesection(DcrcApiPage.java:231)
at com.StudentPortal.StudentTests.StudentsDCRCAPITests.Abilitytoaddacoursesection(StudentsDCRCAPITests.java:343)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- Abilitytoaddacoursesection -->
  <testcase name="CopyClassSchedule" time="1.034" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="CopyClassSchedule" time="1.216" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="CopyClassSchedule" time="1.607" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC898_Addanenrollmenttoastudent" time="3.550" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC898_Addanenrollmenttoastudent" time="2.330" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TC898_Addanenrollmenttoastudent" time="3.773" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="AddacoursetoastudentsscheduleRegister" time="2.129" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.framework.util.TestReportsLog.takeScreenShot(TestReportsLog.java:116)
at com.framework.util.TestReportsLog.log(TestReportsLog.java:41)
at com.StudentPortal.Pages.DcrcApiPage.AddacoursetoastudentsscheduleRegister(DcrcApiPage.java:732)
at com.StudentPortal.StudentTests.StudentsDCRCAPITests.AddacoursetoastudentsscheduleRegister(StudentsDCRCAPITests.java:387)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- AddacoursetoastudentsscheduleRegister -->
  <testcase name="Recordmakeuphoursforastudent" time="0.712" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.framework.util.TestReportsLog.takeScreenShot(TestReportsLog.java:116)
at com.framework.util.TestReportsLog.log(TestReportsLog.java:41)
at com.StudentPortal.Pages.DcrcApiPage.Recordmakeuphoursforastudent(DcrcApiPage.java:776)
at com.StudentPortal.StudentTests.StudentsDCRCAPITests.Recordmakeuphoursforastudent(StudentsDCRCAPITests.java:398)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- Recordmakeuphoursforastudent -->
  <testcase name="Recordmakeuphoursforastudent" time="0.624" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.framework.util.TestReportsLog.takeScreenShot(TestReportsLog.java:116)
at com.framework.util.TestReportsLog.log(TestReportsLog.java:41)
at com.StudentPortal.Pages.DcrcApiPage.Recordmakeuphoursforastudent(DcrcApiPage.java:776)
at com.StudentPortal.StudentTests.StudentsDCRCAPITests.Recordmakeuphoursforastudent(StudentsDCRCAPITests.java:398)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- Recordmakeuphoursforastudent -->
  <testcase name="Recordmakeuphoursforastudent" time="0.771" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.framework.util.TestReportsLog.takeScreenShot(TestReportsLog.java:116)
at com.framework.util.TestReportsLog.log(TestReportsLog.java:41)
at com.StudentPortal.Pages.DcrcApiPage.Recordmakeuphoursforastudent(DcrcApiPage.java:776)
at com.StudentPortal.StudentTests.StudentsDCRCAPITests.Recordmakeuphoursforastudent(StudentsDCRCAPITests.java:398)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- Recordmakeuphoursforastudent -->
  <testcase name="Unregisteracoursefromastudentsschedule" time="0.657" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.framework.util.TestReportsLog.takeScreenShot(TestReportsLog.java:116)
at com.framework.util.TestReportsLog.log(TestReportsLog.java:41)
at com.StudentPortal.Pages.DcrcApiPage.Unregisteracoursefromastudentsschedule(DcrcApiPage.java:833)
at com.StudentPortal.StudentTests.StudentsDCRCAPITests.Unregisteracoursefromastudentsschedule(StudentsDCRCAPITests.java:409)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- Unregisteracoursefromastudentsschedule -->
  <testcase name="AddDegreestoaStudent" time="1.424" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="AddDegreestoaStudent" time="0.623" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="AddDegreestoaStudent" time="3.860" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="AddDegreestoaStudent" time="0.761" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="AddDegreestoaStudent" time="0.678" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="AddHonorstoaStudent" time="0.718" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="AddHonorstoaStudent" time="0.544" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="AddHonorstoaStudent" time="0.691" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="AddHonorstoaStudent" time="0.723" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="AddHonorstoaStudent" time="0.714" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="AddTermHonerandBadges" time="0.735" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="AddTermHonerandBadges" time="0.808" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="AddTermHonerandBadges" time="0.830" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="RecordbatchMidtermGradesforacourse" time="0.711" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="RecordbatchMidtermGradesforacourse" time="0.763" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="RecordbatchMidtermGradesforacourse" time="0.739" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="ConfirmdocumentshavebeencreatedandassociatedwiththeCareerServicesmoduleNewDocument" time="0.730" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="ConfirmdocumentshavebeencreatedandassociatedwiththeCareerServicesmoduleNewDocument" time="0.923" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="ConfirmdocumentshavebeencreatedandassociatedwiththeCareerServicesmoduleNewDocument" time="0.867" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="ConfirmdocumentshavebeencreatedandassociatedwiththeCareerServicesmoduleNewDocumentList" time="0.829" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="ConfirmdocumentshavebeencreatedandassociatedwiththeCareerServicesmoduleNewDocumentList" time="0.728" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="ConfirmdocumentshavebeencreatedandassociatedwiththeCareerServicesmoduleNewDocumentList" time="0.746" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="TasktoconftoallassoevewithStudents" time="0.684" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.framework.util.TestReportsLog.takeScreenShot(TestReportsLog.java:116)
at com.framework.util.TestReportsLog.log(TestReportsLog.java:41)
at com.StudentPortal.Pages.DcrcApiPage.TasktoconftoallassoevewithStudents(DcrcApiPage.java:1295)
at com.StudentPortal.StudentTests.StudentsDCRCAPITests.TasktoconftoallassoevewithStudents(StudentsDCRCAPITests.java:485)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- TasktoconftoallassoevewithStudents -->
  <testcase name="TasktoconftoallassoevewithStudents" time="0.762" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.framework.util.TestReportsLog.takeScreenShot(TestReportsLog.java:116)
at com.framework.util.TestReportsLog.log(TestReportsLog.java:41)
at com.StudentPortal.Pages.DcrcApiPage.TasktoconftoallassoevewithStudents(DcrcApiPage.java:1295)
at com.StudentPortal.StudentTests.StudentsDCRCAPITests.TasktoconftoallassoevewithStudents(StudentsDCRCAPITests.java:485)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- TasktoconftoallassoevewithStudents -->
  <testcase name="TasktoconftoallassoevewithStudents" time="0.812" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.framework.util.TestReportsLog.takeScreenShot(TestReportsLog.java:116)
at com.framework.util.TestReportsLog.log(TestReportsLog.java:41)
at com.StudentPortal.Pages.DcrcApiPage.TasktoconftoallassoevewithStudents(DcrcApiPage.java:1295)
at com.StudentPortal.StudentTests.StudentsDCRCAPITests.TasktoconftoallassoevewithStudents(StudentsDCRCAPITests.java:485)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- TasktoconftoallassoevewithStudents -->
  <testcase name="Conftaskdropdownemployerrecord" time="0.963" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="Conftaskdropdownemployerrecord" time="0.746" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="Conftaskdropdownemployerrecord" time="0.859" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="SubsidiaryLedgerPostaCharge" time="0.626" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="SubsidiaryLedgerPostaCharge" time="0.579" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="SubsidiaryLedgerPostaCharge" time="0.560" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="SubsidiaryLedgerPostaPayment" time="0.834" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="SubsidiaryLedgerPostaPayment" time="0.670" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="SubsidiaryLedgerPostaPayment" time="0.656" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="PerformaProgramTransferforaStudent" time="5.717" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="PerformaProgramTransferforaStudent" time="3.285" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests"/>
  <testcase name="SubsidiaryLedgerAdjustTransactionDelete" time="0.598" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.framework.util.TestReportsLog.takeScreenShot(TestReportsLog.java:116)
at com.framework.util.TestReportsLog.log(TestReportsLog.java:41)
at com.StudentPortal.Pages.DcrcApiPage.SubsidiaryLedgerAdjustTransactionDelete(DcrcApiPage.java:1537)
at com.StudentPortal.StudentTests.StudentsDCRCAPITests.SubsidiaryLedgerAdjustTransactionDelete(StudentsDCRCAPITests.java:540)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- SubsidiaryLedgerAdjustTransactionDelete -->
  <testcase name="SubsidiaryLedgerAdjustTransactionDelete" time="0.735" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.framework.util.TestReportsLog.takeScreenShot(TestReportsLog.java:116)
at com.framework.util.TestReportsLog.log(TestReportsLog.java:41)
at com.StudentPortal.Pages.DcrcApiPage.SubsidiaryLedgerAdjustTransactionDelete(DcrcApiPage.java:1537)
at com.StudentPortal.StudentTests.StudentsDCRCAPITests.SubsidiaryLedgerAdjustTransactionDelete(StudentsDCRCAPITests.java:540)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- SubsidiaryLedgerAdjustTransactionDelete -->
  <testcase name="SubsidiaryLedgerAdjustTransactionDelete" time="0.668" classname="com.StudentPortal.StudentTests.StudentsDCRCAPITests">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.framework.util.TestReportsLog.takeScreenShot(TestReportsLog.java:116)
at com.framework.util.TestReportsLog.log(TestReportsLog.java:41)
at com.StudentPortal.Pages.DcrcApiPage.SubsidiaryLedgerAdjustTransactionDelete(DcrcApiPage.java:1537)
at com.StudentPortal.StudentTests.StudentsDCRCAPITests.SubsidiaryLedgerAdjustTransactionDelete(StudentsDCRCAPITests.java:540)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- SubsidiaryLedgerAdjustTransactionDelete -->
</testsuite> <!-- com.StudentPortal.StudentTests.StudentsDCRCAPITests -->
